projects/plugins/debug-helper/composer.json

```diff
@@ -3,9 +3,9 @@
 	"description": "A plugin for mocking various states of the plugin to aid in Jetpack development. Not for production use.",
 	"type": "wordpress-plugin",
 	"license": "GPL-2.0-or-later",
-	"require": {},
 	"require-dev": {
-		"automattic/jetpack-changelogger": "@dev"
+		"automattic/jetpack-changelogger": "@dev",
+		"phpunit/phpcov": "^10.0"
```

Comment:
I note that putting this in `require-dev` means it won't be included in the download from https://github.com/Automattic/jetpack-debug-helper. I think a user with shell access could do `composer install` to get it into the downloaded copy though.

Since the one use of it seems to be in `scripts/load-coverage-data.php` which looks like a command line script, maybe that's ok. You might want to add a `class_exists()` check and print instructions to do that if the needed classes aren't there.